interface/VideoCommonDefs.h

```diff
@@ -69,6 +69,8 @@ extern "C" {
          ((uint32_t)(uint8_t)(ch2) << 16)  | ((uint32_t)(uint8_t)(ch3) << 24))
 
 #define YAMI_FOURCC_NV12 YAMI_FOURCC('N','V','1','2')
+#define YAMI_FOURCC_P010 YAMI_FOURCC('P','0','1','0')
+#define YAMI_FOURCC_I010 YAMI_FOURCC('I','0','1','0')
```

Comment:
Do we really need this in core library level. Could we support P010 only? And let yami-utils do I010 to P010 conversion?